File: 136 Single Number/136.c
int singleNumber(int* nums, int numsSize) { 
    //若遇只有單個數的陣列直接return    
    if(numsSize == 1){ 
        return *nums; 
    } 
    int i, x = 0; 
    for(i = 0; i< numsSize; i++ ){ 
        //作XOR運算
        x ^= *(nums+i); 
    } 
    return x; 
} 
//Min:8ms

File: 136 Single Number/136.js
/**
 * @param {number[]} nums
 * @return {number} 
 */ 
var singleNumber = function(nums) { 
    if(nums.length == 1){ 
        return nums[0]; 
    } 
    var x=0; 
    for(var i = 0; i < nums.length ; i++){ 
        x = x ^ nums[i]; 
    } 
    return x; 
}; 
//Min:116ms